plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Payments.php

```diff
@@ -115,7 +115,7 @@ function ( $gateway ) {
 	 * @return bool
 	 */
 	public function is_in_progress() {
-		return $this->has_woopayments_test_account() && ! $this->is_complete();
+		return ( $this->has_woopayments_live_account_in_progress() || $this->has_woopayments_test_account() ) && ! $this->is_complete();
```

Comment:
I think we could take the "bail early" approach here since we are gathering up logic. If it is complete, just return false and call it a day.

Reply:
Implemented the early return. Thanks for the suggestion!